arceos-vmm/src/vmm/vcpus.rs

```diff
@@ -281,7 +281,15 @@ fn vcpu_run() {
                         "VM[{}]'s VCpu[{}] try to boot target_cpu [{}] entry_point={:x} arg={:#x}",
                         vm_id, vcpu_id, target_cpu, entry_point, arg
                     );
+                    #[cfg(not(platform = "aarch64-bsta1000b-virt-hv"))]
                     vcpu_on(vm.clone(), target_cpu as _, entry_point, arg as _);
+                    #[cfg(platform = "aarch64-bsta1000b-virt-hv")]
+                    vcpu_on(
+                        vm.clone(),
+                        ((target_cpu / 256) as usize).try_into().unwrap(),
```

Comment:
just, do not hardcode it

since we have `phys_cpu_ids = [0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07]`
 can we use this information stored in vm config?

or we can extend this index array to a key-value type mapping.

Reply:
I see, maybe we can use phys_cpu_ids

Reply:
Since this information is provided by the device tree, we changed the device tree of a1000b in the hypervisor environment, and now it can work normally without hardcode.